src/interpreter.h

```diff
@@ -92,73 +92,71 @@ static const IstreamOffset kInvalidIstreamOffset = ~0;
 #define WABT_TABLE_ENTRY_DROP_OFFSET sizeof(uint32_t)
 #define WABT_TABLE_ENTRY_KEEP_OFFSET (sizeof(IstreamOffset) + sizeof(uint32_t))
 
-enum class InterpreterOpcode {
+enum class Opcode {
```

Comment:
Yes, that is a concern. But I think it's a win when you balance it against having simpler names.